youre-already-a-sponsor/logo-formats-and-media.md

```diff
@@ -1,6 +1,6 @@
 # Logo Formats & Media
 
-All logos have to be in **vector format** \(.ai or .svg\) or if it's not possible, in .png format.
+All logos have to be in **vector format** \(.ai or .svg\) or if it's not possible, in .png format. If we need to add your logo to printed media, it has to be **only in vector format** and in **single color** otherwise it can't be printed.
```

Comment:
```suggestion
All logos have to be in **vector format** \(.ai or .svg\) or if it's not possible, in .png format. If we need to add your logo to printed media, we need a **single color** version **in vector format**, otherwise it can't be printed.
```